open_instruct/reward_modeling.py

```diff
@@ -241,7 +244,8 @@ def main(args: Args, dataset_config: DatasetConfig, model_config: ModelConfig):
         tokenizer.pad_token_id = 128002  # <|reserved_special_token_0|>
     else:
         tokenizer.add_special_tokens({"pad_token": "[PAD]"})  # NOTE: we do not resize the embedding
-    tokenizer.chat_template = CHAT_TEMPLATES[dataset_config.chat_template]
+    if dataset_config.chat_template is not None:
```

Comment:
A little surprised to see this here.

Reply:
Yeah so the logic is if DPO already modifies the chat template, we don't try to modify it again. This is especially important for OLMo with the add_bos baked in to the chat template. 

We will refactor this anyway down the line in #522